src/player.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+class Player {
+
+  constructor(name) {
+    this._name = name;
+    this._score = 0;
+  }
+
+  get name() {
+    return this._name;
+  }
+
+  set name(value) {
+    this._name = value;
+  }
+
+  get score() {
```

Comment:
I would challenge whether its the responsibility of the player class to add up scores... perhaps you need another class?